indra/newview/llpathfindingnavmesh.cpp

```diff
@@ -139,7 +139,7 @@ void LLPathfindingNavMesh::handleNavMeshResult(const LLSD &pContent, U32 pNavMes
         ENavMeshRequestStatus status;
         if ( pContent.has(NAVMESH_DATA_FIELD) )
         {
-            const LLSD::Binary &value = pContent.get(NAVMESH_DATA_FIELD).asBinary();
+            LLSD::Binary value = pContent.get(NAVMESH_DATA_FIELD).asBinary();
```

Comment:
I'd suggest this method to suppress the related compiler warning. It avoids creating an additional copy of the binary data vector.
```suggestion
            LLSD nav_data = pContent.get(NAVMESH_DATA_FIELD);
            const LLSD::Binary& value = nav_data.asBinary();
```

Reply:
Thank you. I refixed it.